core/src/main/java/com/imaginea/kodebeagle/GitHelper/ListFileContents.java

```diff
@@ -0,0 +1,432 @@
+package com.imaginea.kodebeagle.GitHelper;
```

Comment:
"ListFileContents" sounds more like a method name than a class name. IMO, verbs should rarely be used for class names. They should usually be nouns unless good reason otherwise.